src/page-objects/administration/Categories.ts

```diff
@@ -85,6 +85,11 @@ export class Categories implements PageObject {
     this.openInNewTabCheckbox = page.getByRole('checkbox', { name: 'Open in new tab' });
   }
 
+  async openLandingPageByName(landingPageName: string) {
```

Comment:
Honestly I wouldn't add logic to the page-object if it is not necessary. For that we have the task objects. Here we wanna just a Locator logic. Therefore please use the task for `openLandingPageByName()`

Reply:
maybe a task in this case is overkill? I changed to:
`async getLandingPageByName(landingPageName: string) : Promise<Locator> {
    return this.landingPageItems.locator(`text="${landingPageName}"`);
  }`